stacks/spica/src/bucket/pages/add/add.component.html

```diff
@@ -96,41 +96,12 @@ <h6>{{ bucket.description }}</h6>
   </mat-card-content>
 
   <mat-card-actions>
-    <form #scheduleForm="ngForm" *ngIf="data?._schedule">
-      <button [disabled]="bucket.readOnly" mat-button color="warn" (click)="cancelSchedule()">
-        <mat-icon>cancel</mat-icon> Cancel Schedule
-      </button>
-      <mat-form-field>
-        <input
-          matInput
-          readonly
-          required
-          name="schedule_date"
-          [(ngModel)]="data._schedule"
-          [disabled]="bucket.readOnly"
-          [owlDateTime]="dt1"
-          [owlDateTimeTrigger]="dt1"
-          [min]="minScheduleDate"
-        />
-        <owl-date-time #dt1></owl-date-time>
-      </mat-form-field>
-    </form>
-
-    <button
-      *ngIf="!data._schedule"
-      mat-icon-button
-      matTooltip="Set publish date"
-      [disabled]="bucket.readOnly"
-      (click)="schedule()"
-    >
-      <mat-icon>schedule</mat-icon>
```

Comment:
Did we deprecate the schedule feature? If so, I think deprecation should be handled on another task and a new pull request. Is there anything that prevents this separation?